src/infra/execute.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+
+kubectl apply -f pg-secrets.yaml
+kubectl apply -f pg-service.yaml
+kubectl apply -f pg-statefulset.yaml
+
+kubectl apply -f app-configmap.yaml
+kubectl apply -f app-service.yaml
+kubectl apply -f app-deployment.yaml
+
+minikube service tickets-service
```

Comment:
since you are using `minikube` here it's better to name `local.sh` or something like that.

Reply:
fixed c1dcd62